evaluate.sql

```diff
@@ -0,0 +1,247 @@
+--!/usr/bin/env python3
+-- Copyright 2024 Timescale Inc.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- Repository: https://github.com/timescale/migration-eval
+--
+-- This file runs SQL queries against your database and collects necessary data
+-- to generate a migration evaluation report, which allows us to recommend the
+-- most suitable migration strategy for your specific use case.
+--
+-- To run: psql -t -q -d "<POSTGRES_URI>" -f evaluate.sql
+--
+-- The generated report will be printed in the terminal which can be redirected to a file.
+
+\pset tuples_only
+\set ON_ERROR_STOP on
+
+select version() version \gset
+\echo 'PostgreSQL version:' :version
+
+select pg_size_pretty(pg_database_size(current_database())) db_size \gset
+\echo 'Database size:' :db_size
+
+select count(*) from information_schema.tables
```

Comment:
Better give explicit name to the column rather than relying on the system.
```suggestion
select count(*) as count from information_schema.tables
```